.github/MAP_GUIDELINES.md

```diff
@@ -54,3 +54,12 @@ If these features do not work in the current codebase then your map PR will be *
 - Feel free to take inspiration from other maps, but please don't copy paste large parts of them.
 - If you use random item spawners, try using the types of them that create specific numbers of items. This way you don't risk accidentally overloading rooms with items because RNG decided it to be so.
 - Spacemas ornaments are now automatically generated as appropriate. You must only manually add the tree somewhere, specifically the ephemeral subtype.
+
+## Tips and Tricks
+###### These aren't one hundred percent necessary to make maps, but they might make your life a bit easier
+- **Place Directions!** They make new maps a lot easier to navigate, not just for players, but for you as well during the process. Use the donut 2/nadir floor decals, or the wall signs like cogmap 1, or even both. The amount of directions you should place should be proportional to map size, hallway size, and mazelikeness.
```

Comment:
I think there was a dislike of the current design of these floor decals from the dev team. @flrsh ?

Reply:
Well, I like them :(

Reply:
> a dislike of the current design of these floor decals

echo

Reply:
I personally like the floor decals if used sparingly. Wall decals, if you can find the space, are always a nice touch.

Reply:
Yeah I don't like them and don't want to encourage using them in the official map guidelines. Wall decals are fine though!

Reply:
going to remove all mention of the floor decals, and recommend wall decals only 👍